tensorboard/plugins/scalar/scalars_plugin.py

```diff
@@ -127,7 +121,7 @@ def index_impl(self, experiment=None):
 
         return result
 
-    def scalars_impl(self, tag, run, experiment, output_format):
+    def scalars_impl(self, tag, run, experiment):
```

Comment:
Looks like legit test failure in the custom scalars plugin due to the
removal of this parameter.

Reply:
Thanks, will fix shortly.